modules/webxr/config.py

```diff
@@ -0,0 +1,15 @@
+def can_build(env, platform):
+    return True
```

Comment:
Make sure you check your platform here so the module only gets compiled for HTML5

Reply:
Fixed in latest commit!

Reply:
Ended up going back to the original here, so that we could compile in the abstract class and get autocomplete in the editor.